packages/fluentui/react-northstar/src/components/Popup/Popup.tsx

```diff
@@ -439,7 +439,12 @@ export const Popup: React.FC<PopupProps> &
 
             {context.target && (
               <>
-                <EventListener listener={handleDocumentClick(getRefs)} target={context.target} type="click" capture />
+                <EventListener
+                  listener={handleDocumentClick(getRefs)}
+                  target={context.target}
+                  type="mousedown"
```

Comment:
it's an interesting solution, you might have to modify tests since, generally tests will only trigger the specific event.

Perhaps this should also be brought up in open hours as an FYI.